src/services/userUpdated.js

```diff
@@ -0,0 +1,17 @@
+import User from "../models/user";
+
+const userUpdated = async (req) => {
+    const user = await User.findOneAndUpdate(
```

Comment:
An unhandled error occurs if a user doesn't exist. You could consider wrapping the function in `try ... catch ...` with catch being for instance:
```
catch {
    return { status: 'invalid', message: 'User not found' };
  }
```